public/locales/en-US/translations.json

```diff
@@ -200,7 +200,7 @@
   "invalid_transaction_hash": "The transaction hash is invalid",
   "ledger_not_found": "Ledger not Found",
   "check_ledger_id": "Please check your ledger id",
-  "server_ledgers_hint": "This node ({{connection.server.publicKey, truncate(length: 10)}}) only contains ledgers {{connection.ledger.validated}}",
```

Comment:
I think we still want the pubkey to be printed if it's available, with some sort of "This Clio node only..." alternate message

Reply:
So, you're saying that the **clio-node-part** still needs to be present in the code. What if I change the code such that whenever the **pubKey** is not empty, it gets displayed. But when it is empty, print it as "This clio node only..."

Would that be viable?

Reply:
Yes, something like that sounds good.